PurchasesTests/Networking/HTTPClientTests.swift

```diff
@@ -825,7 +825,7 @@ private extension HTTPClientTests {
             XCTFail(error.localizedDescription)
         }
 
-        return -999
+        fatalError("Unreachable")
```

Comment:
Maybe put this inside the `catch` to make it more obvious that the other branch already returns something, and this is only to deal with `XCTFail` not being marked as unreachable?

Reply:
🤔 yeah, it makes sense. Also, I've used a specific error message, because the captured error didn't provide a clear context. 

`No se pudo completar la operación. (Error de com.apple.XCTestErrorDomain 105)`